### PR TITLE
[208483] Test rdi kobo

### PR DESCRIPTION
**Pull request by patryk-dabrowski**
[AB#208483](https://unicef.visualstudio.com/4e044e8d-bc28-4768-8074-f80ff6e4a0e5/_workitems/edit/208483): RDI import error test

### CODE REVIEW

**Review by pavlo-mk — APPROVED** [review]

File: backend/selenium_tests/registration_data_import/test_registration_data_import.py [hunk 1]
```diff
@@ -174,3 +176,87 @@ def test_smoke_registration_data_import_happy_path(
...
+        assert pageRegistrationDataImport.titleText in pageRegistrationDataImport.getPageHeaderTitle().text
+        pageRegistrationDataImport.getButtonImport().click()
+        # Check Elements on Page
+        assert pageRegistrationDataImport.getButtonImportFile().get_property("disabled")
+        pageRegistrationDataImport.getImportTypeSelect().click()
+        assert pageRegistrationDataImport.koboItemText in pageRegistrationDataImport.getKoboItem().text
+        pageRegistrationDataImport.getKoboItem().click()
+        pageRegistrationDataImport.getInputName().send_keys("Test 1234 !")
+
+        # pageRegistrationDataImport.checkLoadingProgressCircle()
```
Comment (szymon-kellton) [inline backend/selenium_tests/registration_data_import/test_registration_data_import.py:249]: Please delete unnecessary comments?

File: backend/selenium_tests/registration_data_import/test_registration_data_import.py [hunk 2]
```diff
@@ -174,3 +176,87 @@ def test_smoke_registration_data_import_happy_path(
...
+        pageRegistrationDataImport.getKoboProjectSelect().click()
+        pageRegistrationDataImport.select_listbox_element("UNICEF NGA Education").click()
+        # pageRegistrationDataImport.checkLoadingProgressCircle()
```
Comment (szymon-kellton) [inline backend/selenium_tests/registration_data_import/test_registration_data_import.py:252]: Please delete unnecessary comments?

File: backend/selenium_tests/registration_data_import/test_registration_data_import.py [hunk 3]
```diff
@@ -174,3 +176,87 @@ def test_smoke_registration_data_import_happy_path(
...
+        business_area.countries.set([country])
+
+        area_type_1 = AreaType.objects.create(name="State", area_level=1, country=country)
+        area_type_2 = AreaType.objects.create(
+            name="Local government area", area_level=2, country=country, parent=area_type_1
+        )
+        area_type_3 = AreaType.objects.create(name="Ward", area_level=3, country=country, parent=area_type_2)
+        area_1 = Area.objects.create(name="Borno", p_code="NG008", area_type=area_type_1)
+        area_2 = Area.objects.create(name="Bama", p_code="NG008003", area_type=area_type_2, parent=area_1)
+        Area.objects.create(name="Andara", p_code="NG008003001", area_type=area_type_3, parent=area_2)
```
Comment (szymon-kellton) [inline backend/selenium_tests/registration_data_import/test_registration_data_import.py:235]: From 
business_area = BusinessArea.objects.get(name="Afghanistan") 
to
Area.objects.create(name="Andara", p_code="NG008003001", area_type=area_type_3, parent=area_2)

From my perspective that should be in separate pytest fixture/fixtures. Consider for test clearance.

File: backend/selenium_tests/registration_data_import/test_registration_data_import.py [hunk 4]
```diff
@@ -174,3 +176,87 @@ def test_smoke_registration_data_import_happy_path(
...
+        assert pageRegistrationDataImport.titleText in pageRegistrationDataImport.getPageHeaderTitle().text
+        pageRegistrationDataImport.getButtonImport().click()
+        # Check Elements on Page
+        assert pageRegistrationDataImport.getButtonImportFile().get_property("disabled")
+        pageRegistrationDataImport.getImportTypeSelect().click()
+        assert pageRegistrationDataImport.koboItemText in pageRegistrationDataImport.getKoboItem().text
+        pageRegistrationDataImport.getKoboItem().click()
+        pageRegistrationDataImport.getInputName().send_keys("Test 1234 !")
+
+        # pageRegistrationDataImport.checkLoadingProgressCircle()
```
Comment (szymon-kellton) [inline backend/selenium_tests/registration_data_import/test_registration_data_import.py:201]: Please delete unnecessary comments?

File: backend/selenium_tests/registration_data_import/test_registration_data_import.py [hunk 5]
```diff
@@ -174,3 +176,87 @@ def test_smoke_registration_data_import_happy_path(
...
+        pageRegistrationDataImport.getKoboProjectSelect().click()
+        pageRegistrationDataImport.select_listbox_element("Education new programme").click()
+        # pageRegistrationDataImport.checkLoadingProgressCircle()
```
Comment (szymon-kellton) [inline backend/selenium_tests/registration_data_import/test_registration_data_import.py:204]: Please delete unnecessary comments?

**Review by szymon-kellton — CHANGES_REQUESTED** [review]

**Review by szymon-kellton — APPROVED** [review]

**Pull request merged**